src/otx/v2/adapters/datumaro/adapter/datumaro_dataset_adapter.py

```diff
@@ -219,6 +219,8 @@ def get_otx_dataset(self) -> dict[Subset, DatumDataset]:
 
     def get_label_schema(self) -> LabelSchemaEntity:
         """Get Label Schema."""
+        label_information = self._prepare_label_information(self.dataset)
+        self.label_entities = label_information["label_entities"]
```

Comment:
In my opinion, `get_something()` method should give something, but nothing else. Setting `self.label_entities` in this method is a side-effect, which is counterintuitive.
Why don't you make it more explicit?

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2631/commits/3ca5154cc9fd8172865590ad46d03db01b772a91